examples/http/opnsense_services.js

```diff
@@ -11,6 +11,10 @@
  *
  **/
 
+// serviceName: Set it to 'ALL' to retrieve information for all services,
+// or specify a list of services to filter and display only the selected ones.
+var serviceName = D.getParameter("serviceName");
```

Comment:
This is supposed to be a list of service names. At least make sure the variable name is clear enough